mrbeam_ledstrips/state_animations.py

```diff
@@ -1131,11 +1131,14 @@ def _get_frame_duration(self, fps):
 
 
 	def _get_int_val(self, value):
+		# Implements a last_valid_progress value to avoid wrong progress values
 		try:
 			value = int(float(value))
+			self.last_valid_progress=value
```

Comment:
```suggestion
                         # Implements a last_valid_progress value to avoid wrong progress values
			self.last_valid_progress=value
```